packages/@aws-cdk/aws-kinesis/README.md

```diff
@@ -48,3 +48,24 @@ const stream = new Stream(this, 'MyEncryptedStream', {
 
 assert(stream.encryptionKey === myKmsKey);
 ```
+
+### Granting IAM Permissions on Stream to Grantable
+
+Read and Write iam actions can be added to a iam.IGrantable. If the stream has an encryption key attributed, access 
+to encrypt and/or decrypt with the encryption key will also be added to the grantable. There are three instance methods 
+available on a Stream object:  
+*.grantWrite(grantee: iam.IGrantable)  
```

Comment:
Notice the formatting here is not correct, you need a space after the `*`. Have a look at the [rendered](https://github.com/reillykw/aws-cdk/tree/kinesis-stream-permissions-list-shards/packages/%40aws-cdk/aws-kinesis#granting-iam-permissions-on-stream-to-grantable) version, you'll see where the formatting fails.

Also, thats not exactly the content I had in mind. Here, you dont need to list the methods that a `Stream` exposes, we have API docs for that.

Reply:
updated